SPECS/keras/keras.spec

```diff
@@ -0,0 +1,74 @@
+%define _enable_debug_package 0
+%global debug_package %{nil}
+Summary:        Keras is a high-level neural networks API, written in Python and capable of running on top of TensorFlow, CNTK, or Theano.
+Name:           keras
+Version:        2.11.0
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages/Python
+URL:            https://keras.io/
+Source0:        https://github.com/keras-team/keras/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Source1:        %{name}-%{version}-cache.tar.gz
+BuildRequires:  bazel = 5.3.0
+BuildRequires:  build-essential
+BuildRequires:  git
+BuildRequires:  libstdc++-devel
+BuildRequires:  pyproject-rpm-macros
+BuildRequires:  python3-devel
+BuildRequires:  python3-numpy
+BuildRequires:  python3-packaging
+BuildRequires:  python3-pip
+BuildRequires:  python3-requests
+BuildRequires:  python3-wheel
+BuildRequires:  tar
+BuildRequires:  which
+BuildRequires:  python3-tf-nightly = 2.11.0
+ExclusiveArch:  x86_64
+
+%description
+Keras is a deep learning API written in Python, running on top of the machine learning platform TensorFlow. 
+
+%package -n     python3-keras
+Summary:        python-keras
+
+
+%description -n python3-keras
+Python 3 version.
+
+%prep
+%autosetup -p1
+
+
+%build
+# tar -xf %{SOURCE1} -C /root/
```

Comment:
Dead code?  Probably should remove

Reply:
Alternatively I'd double-check we're using the cache correctly, because at the moment `Source1` is not used anywhere in this spec. If we don't need the cache to build the package (keep in mind that test runs enable network access, so they might pass, because Bazel is able to download other dependencies), let's remove the cache. If we do need it, I'm guessing we need this line uncommented.